#CustomBatteryState.msg

float32 voltage 
float32 current 
float32 percentage

float32 charge_current
float32 discharge_current
                
float32 min_cell_voltage
float32 max_cell_voltage
float32 min_cell_temperature
float32 max_cell_temperature
bool allow_charge
bool allow_discharge
